internal/pkg/container/config.go

```diff
@@ -27,3 +28,19 @@ func ImageParentDir() string {
 func ImageFile(name string) string {
 	return path.Join(ImageParentDir(), name+".img")
 }
+
+// InitramfsBootPath returns the dracut built initramfs path, as dracut built initramfs inside container
+// the function returns host path of the built file
+func InitramfsBootPath(image, kver string) (image_path string) {
+	image_path = path.Join(RootFsDir(image), "boot", "initramfs-"+kver+".img")
+	if util.IsFile(image_path) {
+		return image_path
+	}
+
+	image_path = path.Join(RootFsDir(image), "boot", "initrd-"+kver)
+	if util.IsFile(image_path) {
+		return image_path
+	}
+
+	return ""
+}
```

Comment:
@JasonYangShadow I had to add an additional search pattern for Suse. I'm not super happy with this implementation, so feel free to improve it.

I squashed this into your commit, so it's still attributed to you. Let me know if you have any concerns.

Reply:
I've made some changes in this commit
https://github.com/warewulf/warewulf/pull/1191/commits/75c70fe3e2893808f9bff97049a1cb1b434c55b2

Reply:
@JasonYangShadow I changed it a bit against to more closely match what's in `internal/pkg/kernel/kernel.go`.

I also removed a test case that dependendent on the order of files being returned, as I'd consider that undefined. (I know the golang Walk makes it deterministic, but I'd rather that not be an assumption.